package.json

```diff
@@ -31,16 +31,22 @@
   "engines": {
     "node": ">=4.0.0"
   },
+  "main": "./dist/index.js",
+  "typings": "./dist/index.d.ts",
   "files": [
     "CHANGELOG.md",
     "LICENSE",
     "README.md",
-    "index.js",
-    "lib/"
+    "dist/index.js",
+    "dist/index.d.ts"
```

Comment:
I don't think these are the only files that'll be needed. `dist/index.js` requires several other files, which I think will be missed here.

Could you make sure all of the required files are published, either by (1) dumping everything into `dist/index.js` (2) making sure the appropriate files are listed here?

You can test this by running `npm pack`, which should generate something like `helmet-csp-2.8.0.tgz`. Then, from another package, running `npm install /path/to/helmet-csp-2.8.0.tgz`, and making sure it works.

Reply:
Good point - I've added the `dist` folder which I think is probably simpler then listing each file - let me know if you'd rather list each file instead.